network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -155,6 +155,13 @@ class BlogIndexPage(IndexPage):
     def get_context(self, request):
         context = super().get_context(request)
         context["related_topics"] = self.get_related_topics()
+
+        if not hasattr(self, 'filtered') and self.page_size == 12 or self.page_size == 24:
```

Comment:
Making sure we are not on a filtered page, which do not render the callout box at all.

Also making sure that the page size is 12/24, which are the only page sizes divisible by 3 and having this issue.

Reply:
It would be good to move this check into a helper method that clearly states what this check is supposed to mean. Something like `is_showing_callout_box_in_entries`. 

Also, we probably don't want to show the callout box if there are not topics.

Reply:
Great Idea! Updated and also added the check for the topics